cdap-docs/user-guide/source/data-preparation/directives/parse-timestamp.rst

```diff
@@ -0,0 +1,33 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2018 Cask Data, Inc.
+
+===============
+Parse Timestamp
+===============
+
+The PARSE-TIMESTAMP directive parses column values representing unix timestamp as date.
```

Comment:
The PARSE-TIMESTAMP directive parses column values that represent a Unix timestamp.